e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy/change-your-answers/loss-payee/change-your-answers-change-loss-payee-details-uk-name.spec.js

```diff
@@ -0,0 +1,91 @@
+import partials from '../../../../../../../../partials';
+import { field, summaryList } from '../../../../../../../../pages/shared';
+import { POLICY as POLICY_FIELD_IDS } from '../../../../../../../../constants/field-ids/insurance/policy';
+import { INSURANCE_ROUTES } from '../../../../../../../../constants/routes/insurance';
+import application from '../../../../../../../../fixtures/application';
+
+const {
+  LOSS_PAYEE_DETAILS: { NAME },
+} = POLICY_FIELD_IDS;
+
+const {
+  ROOT,
+  POLICY: { LOSS_PAYEE_DETAILS_CHECK_AND_CHANGE },
+  CHECK_YOUR_ANSWERS: { TYPE_OF_POLICY },
+} = INSURANCE_ROUTES;
+
+const fieldId = NAME;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.submitApplication.tasks.checkAnswers;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context(`Insurance - Change your answers - Policy - - Loss payee details - UK - ${NAME} - As an exporter, I want to change my answers to the loss payee section`, () => {
```

Comment:
typo,  2 -

Reply:
Whoops, good spot - will fix in the next PR